### PR TITLE
File structure improved, with some additional adjustments (Code refactoring)

### PR DESCRIPTION
**Pull request by KrystianZybura**
I made some changes in our data organization in our project, improved readability of the file structures with some additional improvements in the code, especially in the features. Obviously, if you have any further improvement ideas or those changes aren't satisfying, I'm fully open to your improvement ideas 👌

### CODE REVIEW

**Review by WoytekMig — APPROVED** [review]
Good work - everything seems fine, only one thing maybe -  folder name 'helpers' ? ;) those files could be just in 'common' folder, or maybe in two : API (for codesAPI and Genres) and Page (for usePage...) :) 
That's my suggestion, but I don't force it ;)

File: src/App/features/Details/MovieDetails/MovieInfo/styled.js [hunk 1]
```diff
@@ -6,15 +6,9 @@ export const Wrapper = styled.div`
...
   grid-template-areas:
     "pic data"
-    "pic desc"
-    "pic desc"
-    "pic desc"
-    "pic desc"
-    "pic desc"
-    "pic desc"
-    "pic desc"
     "pic desc";
```
Comment (Arcziwald) [inline src/App/features/Details/MovieDetails/MovieInfo/styled.js:9]: Good idea :-)

**Review by Arcziwald — APPROVED** [review]
You handled it great :-)

**Comment by KrystianZybura** [comment]
> Good work - everything seems fine, only one thing maybe - folder name 'helpers' ? ;) those files could be just in 'common' folder, or maybe in two : API (for codesAPI and Genres) and Page (for usePage...) :) That's my suggestion, but I'm don't force it ;)

I honestly think the folder named "Helpers" speaks for itself and contains all the files that are used in multiple places, but aren't react component files. I agree with you with the genreIds.js file, which should be in MovieTile folder, because it's only being used in that component, so I'm going to move it there, good call ✌

**Pull request merged**